### PR TITLE
feat: remove visible prop for Tag

### PR DESCRIPTION
**Pull request by yykoypj**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [x] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

https://github.com/ant-design/ant-design/issues/36609#issuecomment-1189845088

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | remove visible props of Tag          |
| 🇨🇳 Chinese | 移除Tag组件的visible属性          |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
用户如果使用了 visible ，可以 warning 提示一下。

**Comment by yykoypj** [comment]
> 用户如果使用了 visible ，可以 warning 提示一下。

Done.

File: components/tag/index.tsx [hunk 1]
```diff
@@ -102,6 +103,12 @@ const InternalTag: React.ForwardRefRenderFunction<HTMLSpanElement, TagProps> = (
     return null;
   };
 
+  warning(
+    !('visible' in props),
+    'Tag',
+    '`visible` is deprecated, you can use `visible && <Tag />` instead.',
```
Comment (afc163) [inline components/tag/index.tsx:109]:
```suggestion
    '`visible` is removed, you can use `visible && <Tag />` instead.',
```

**Comment by zombieJ** [comment]
在 feature 上搞一个 deprecated warning 出来提示一下用户一些方法在 next major 要废弃了~

**Pull request merged**